pkg/storage/etcd/etcd_helper_test.go

```diff
@@ -352,28 +352,11 @@ func TestSetWithVersion(t *testing.T) {
 	}
 }
 
-func TestSetWithoutResourceVersioner(t *testing.T) {
```

Comment:
Yes - as I wrote in another comment, I currently need to have versioner to clean the version. We need to decide whether we want to enforce having a versioner or just accept the fact that if we don't have versioner we can have resourceVersion stored in etcd. I think the former is better...
@smarterclayton @lavalamp for thoughts